### PR TITLE
Upgrade storybook to 7

### PR DESCRIPTION
**Pull request by alantoa**
# Why

- Upgrade storybook to 7.
- Clear the storybook project and remove unuseful files.
- Make `next/image` to supports storybook.

# How

# Test Plan

-

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@v2
         with:
-          node-version: 14.x
+          node-version: 16.x
```
Comment (alantoa) [inline .github/workflows/chromatic.yml:22]: new `chromatic` version require node16.

File: apps/next/tailwind.config.js [hunk 2]
```diff
@@ -102,13 +102,9 @@ module.exports = {
         ".text-3xl": textSizes["text-3xl"],
         ".text-4xl": textSizes["text-4xl"],
         ".safe-top": {
-          paddingTop: "constant(safe-area-inset-top)",
-          // eslint-disable-next-line no-dupe-keys
           paddingTop: "env(safe-area-inset-top)",
         },
         ".safe-bottom": {
-          paddingBottom: "constant(safe-area-inset-bottom)",
```
Comment (alantoa) [inline apps/next/tailwind.config.js:110]: remove this because `constant` only support iOS <= `11.2`, but now nobody is using iOS 11 basically

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️

File: apps/next/tailwind.config.js (hunk 2 above)
Comment (hirbod) [reply apps/next/tailwind.config.js:110]: Yes, that is fine.

**Review by hirbod — APPROVED** [review]
LGTM

**Pull request merged**